README.md

```diff
@@ -38,17 +38,17 @@ around 40% of the code size.
 to execute a given smart contract. Also contains code for gas metering, storing, and caching wasm artifacts. Read more [here](https://github.com/confio/cosmwasm/blob/master/lib/vm/README.md).
 * [go-cosmwasm](https://github.com/confio/go-cosmwasm) - High-level go bindings to all the power inside `cosmwasm-vm`. Easily allows you to upload, instantiate and execute contracts,
 making use of all the optimizations and caching available inside `cosmwasm-vm`.
-* [Cosmos SDK](https://github.com/cosmwasm/modules/tree/master/incubator/contract) - Currently an WIP fork targeting `cosmos/modules` 
-to provide an wasm module you can easily plug into any Cosmos-SDK based application. 
- 
+* [Cosmos SDK](https://github.com/cosmwasm/modules/tree/master/incubator/contract) - Currently an WIP fork targeting `cosmos/modules`
```

Comment:
This is out of date - we need to point to github.com/cosmwasm/wasmd

Reply:
README updated